daemon/api.go

```diff
@@ -770,29 +769,35 @@ func sideloadSnap(c *Command, r *http.Request) Response {
 
 	// form.File is a map of arrays of *FileHeader things
 	// we just allow one (for now at least)
+	var snapBody multipart.File
 out:
 	for _, v := range form.File {
```

Comment:
Updated to initialize origPath from the filename in the form. Will need to move the variable declaration before the loop.